pkg/deploy/upgrade.go

```diff
@@ -39,9 +42,21 @@ func (d *deployer) SaveVersion(ctx context.Context) error {
 	blobClient := azstorage.NewAccountSASClient(
 		*d.config.Configuration.RPVersionStorageAccountName, v, (*d.env.Environment()).Environment).GetBlobService()
 
+	// save version of RP which is deployed in this location
 	containerRef := blobClient.GetContainerReference("rpversion")
-
-	// save rpVersion deployed to current location
 	blobRef := containerRef.GetBlobReference(d.config.Location)
-	return blobRef.CreateBlockBlobFromReader(bytes.NewReader([]byte(d.version)), nil)
+	err = blobRef.CreateBlockBlobFromReader(bytes.NewReader([]byte(d.version)), nil)
+	if err != nil {
+		return err
+	}
+
+	// save OCP streams which are used by RP in this location
+	containerRef = blobClient.GetContainerReference("ocpversions")
+	blobRef = containerRef.GetBlobReference(d.config.Location)
+	ocpVersions := []string{}
+	for _, s := range version.Streams {
+		ocpVersions = append(ocpVersions, (*s.Version).String())
+	}
+	ocpVersionsString := strings.Join(ocpVersions, ",")
+	return blobRef.CreateBlockBlobFromReader(bytes.NewReader([]byte(ocpVersionsString)), nil)
```

Comment:
Would it not be better to store some json? Will make easier to read out?

Reply:
That's fine with me. I'll change it.

Reply:
It will be easier for later when we do multi-version installs :)

Reply:
WDYT about something like this?

```
{
    "installStreams": [
        "4.7.21"
    ],
    "upgradeStreams": [
        "4.7.21",
        "4.6.40",
        "4.5.39",
        "4.4.33"
    ]
}
```

Reply:
Would we want to include PullSpec in there as well just in case it's useful in the future?

Reply:
@mjudeikis Updated to produce following JSON (formatted below for legibility):

```
[
    {
        "version": "4.7.21",
        "pullSpec": "quay.io/openshift-release-dev/ocp-release@sha256:60454c3270a61432447c752a9d5ee3e7d84cebcfd5d371eb4daa263623923dac"
    },
    {
        "version": "4.6.40",
        "pullSpec": "quay.io/openshift-release-dev/ocp-release@sha256:a8bc2a472611d499d99f347b3e2a7c385107e9c4e44b45d765a99338b566dd12"
    },
    {
        "version": "4.5.39",
        "pullSpec": "quay.io/openshift-release-dev/ocp-release@sha256:c4b9eb565c64df97afe7841bbcc0469daec7973e46ae588739cc30ea9062172b"
    },
    {
        "version": "4.4.33",
        "pullSpec": "quay.io/openshift-release-dev/ocp-release@sha256:a035dddd8a5e5c99484138951ef4aba021799b77eb9046f683a5466c23717738"
    }
]
```

Reply:
I added a separate commit which renames `Streams` to `UpgradeStreams` to make its purpose clearer going forward. Once we allow multiple install versions, `UpgradeStreams` will ∌ `InstallStreams`, since there may be multiple z versions in `InstallStreams`.